pom.xml

```diff
@@ -37,7 +50,7 @@
         <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
         <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
         <quarkus.platform.version>3.2.0.Final</quarkus.platform.version>
-        <htmlunit.version>2.70.0</htmlunit.version>
+        <htmlunit.version>3.4.0-SNAPSHOT</htmlunit.version>
```

Comment:
Should we use the SNAPSHOT or instead wait till it is released officially ?

Reply:
No, I don't think so. It is why I created the PR in Draft mode, when the release will be done, I will update the good version and convert the PR to a definitive one